osbuild/main_cli.py

```diff
@@ -163,7 +163,8 @@ def osbuild_cli() -> int:
     monitor_name = args.monitor
     if not monitor_name:
         monitor_name = "NullMonitor" if args.json else "LogMonitor"
-    monitor = osbuild.monitor.make(monitor_name, args.monitor_fd)
+    monitor = osbuild.monitor.make(monitor_name, args.monitor_fd, manifest)
+    monitor.log(f"starting {args.manifest_path}", origin="org.osbuild.main")
```

Comment:
I dislike the `org.osbuild.main` notation here; this doesn't come from an osbuild module, let's call it `osbuild.main_cli` after the Python module name. Same goes for the following few calls.